examples/braket_features/Using_rigetti_calibration_data_in_noise_channel_parameters/Using_rigetti_calibration_data_in_noise_channel_parameters.ipynb

```diff
@@ -0,0 +1,1151 @@
+{
```

Comment:
<p>Could use some code comments since you're doing a lot here that's different from the previous functions and w/o explanation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='73'/>